InvenTree/plugin/helpers.py

```diff
@@ -156,9 +164,9 @@ def get_modules(pkg, path=None):
     elif type(path) is not list:
         path = [path]
 
-    for loader, name, _ in pkgutil.walk_packages(path):
+    for finder, name, _ in pkgutil.walk_packages(path):
         try:
-            module = loader.find_module(name).load_module(name)
+            module = module_from_spec(finder.find_spec(name))
```

Comment:
This line change is the cause of issues here - not sure exactly why yet. Reverting just this line results in the tests passing OK